tests/core/test_broker.py

```diff
@@ -96,6 +99,18 @@ def test_pika_queues_from_args(
     assert pika_processor.queues == expected
 
 
+async def test_pika_raise_rasa_exception(monkeypatch: MonkeyPatch):
+
+    monkeypatch.setattr(
+        PikaEventBroker, "connect", AsyncMock(side_effect=ChannelNotFoundEntity())
+    )
+
+    with pytest.raises(RasaException):
```

Comment:
This works because `RasaException` is a subclass of `ConnectionException`, but maybe better to test that `ConnectionException` is raised?

Reply:
Actually, `ConnectionException` is a subclass of `RasaException`, I've tested locally with `ConnectionException` - all seems fine so I'll make this edit.

Reply:
> Actually, ConnectionException is a subclass of RasaException
Yup sorry that's what I meant - wouldn't work if it was the other way around.